velox/expression/VectorFunction.h

```diff
@@ -175,6 +175,13 @@ class SimpleFunctionAdapterFactory {
 std::optional<std::vector<FunctionSignaturePtr>> getVectorFunctionSignatures(
     const std::string& name);
 
+/// Returns a pair of a list of signatures supported by VectorFunction and the
+/// function metadata with the specified name. Returns std::nullopt if there is
+/// no function with the specified name.
+std::optional<
+    std::pair<std::vector<FunctionSignaturePtr>, VectorFunctionMetadata>>
+getVectorFunctionSignaturesAndMetadata(const std::string& name);
```

Comment:
Looks like the only caller of this API uses only the metadata. Perhaps, change the API to return just that:

```
std::optional<VectorFunctionMetadata> getVectorFunctionMetadata(const std::string& name);
```

Reply:
Updated. Thanks.